src/sage/schemes/riemann_surfaces/riemann_surface.py

```diff
@@ -312,11 +312,16 @@ def differential_basis_baker(f):
     that is nonsingular outside the coordinate points at infinity
     (1:0:0), (0:1:0), and (0:0:1).
 
-    This method applies Baker's theorem, which states that if a curve's
-    singularities are restricted to the coordinate vertices in the projective
-    plane and it meets certain easily testable genericity criteria, a basis
-    for the regular differentials can be derived from the interior integral
-    points of its Newton polygon.
+    Baker's theorem tells us that if a curve has its singularities at 
```

Comment:
Some checks here failing because of trailing whitespace, this should be rectified.